Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -330,3 +372,26 @@ private func applyIncrementDateLabelStyle(_ label: UILabel) {
   label.textAlignment = .left
   label.adjustsFontForContentSizeCategory = true
 }
+
+private func applyIneligibleBadgeViewStyle(_ view: UIView) {
+  view.backgroundColor = .ksr_support_100
+  view.rounded(with: Styles.grid(1))
+}
+
+private func applyIneligibleBadgeLabelStyle(_ label: UILabel) {
+  label.font = UIFont.ksr_caption1().bolded
+  label.textColor = .ksr_support_500
+  label.textAlignment = .center
+  label.numberOfLines = 0
+  label.adjustsFontForContentSizeCategory = true
+}
+
+private func applyTextColorByState(_ label: UILabel, isEnabled: Bool) {
+  var textColor = UIColor.ksr_black
+
+  if !isEnabled {
+    textColor = .ksr_support_300
+  }
+
+  label.textColor = textColor
```

Comment:
nit: this could be `label.textColor = isEnabled ? .ksr_black : .ksr_support_300`